functions/wcif.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs')
+
+const ClearWCIF = {
+  name: 'ClearWCIF',
+  // FIXME: quick helper to make sure we start "from scratch", maybe also
+  // clean other properties/extensions which I'm not aware of?
+  docs: 'Remove all childActivities keeping only the main schedule, remove all assignments.',
+  args: [],
+  outputType: 'Void',
+  usesContext: true,
+  mutations: ['schedule', 'persons'],
+  implementation: (ctx, filename) => {
+    ctx.competition.persons.forEach((person) => {
+      // FIXME: should we clean up "roles" too actually?
+      // Changing "delegate"/"organizers"/"trainee-delegate" would have no effect
+      // as they are - expectedly! - not patchable on the WCA, but any "staff-*"
+      // should likely be removed too.
+      person.assignments = []
+    })
+    ctx.competition.schedule.venues.forEach((venue) => {
+      venue.rooms.forEach((room) => {
+        room.extensions = []
+        room.activities.forEach((activity) => {
+          activity.extensions = []
+          activity.childActivities = []
+        })
+      })
+    })
+  }
+}
+
+const ImportWCIF = {
+  name: 'ImportWCIF',
+  docs: 'Import the WCIF from a json file',
+  args: [
+    {
+      name: 'filename',
+      type: 'String',
+      docs: 'WCIF filename (relative to SCRIPT_BASE)',
+      defaultValue: 'input.json',
+    },
+  ],
+  outputType: 'String',
+  usesContext: true,
+  implementation: (ctx, filename) => {
+    var fullPath = `${process.env.SCRIPT_BASE || '.'}/${filename}`
```

Comment:
What do you think about adding the competitionId in the path? That way you can make sure you don't accidentally import from a different competition.

Reply:
I knew I had forgotten something; I had a slightly different but arguably "better" idea: make sure the imported competition has the same id as the competition we're currently working on!
Sounds good?

Reply:
I'm not a big fan of imposing a directory structure in a context where I'd personally just point `SCRIPT_BASE` to a git repo containing the scripts for a specific competition, but I went ahead and implemented both the id check and the inclusion of the competition id in the path (and I also updated the documentation to clarify this).